src/api/config/routes.rb

```diff
@@ -30,6 +30,7 @@ def self.public_or_about_path?(request)
     binary:       %r{[^\/]*},
     filename:     %r{[^\/]*},
     id:           %r{\d*},
+    session:      %r{[^\/]*},
     login:        %r{[^\/]*},
```

Comment:
Note that I also restored, this constraint for the same reason that `get` login/logout routes. Is it right?

Reply:
@dgdavid This would only be needed if there would be a parameter named ` session `

Reply:
Ups! :innocent: Thanks for clarification. I am going to remove it.